app/views/finders/_result_count.mustache

```diff
@@ -1,3 +1,5 @@
+<h2 class="govuk-body">
```

Comment:
On the herokuapp, it looks as though the result count is wrapped by 2 identical `<h2 class="govuk-body">` tags.

Reply:
Good spot. This should now be fixed as a result of some refactoring that I did